skyportal/models.py

```diff
@@ -74,12 +75,25 @@ class Group(Base):
     photometry = relationship("Photometry", secondary="group_photometry",
                               back_populates="groups",
                               cascade="save-update, merge, refresh-expire, expunge")
+    single_user_group = sa.Column(sa.Boolean, default=False)
```

Comment:
I think this should be excluded from the marshmallow schema

Reply:
@dannygoldstein this is something we will likely use on the front-end, thus it's an expected part of the API call return values and should be left in the schema.

Reply:
I think this is fine to include for GETs but not something we should include in POST

Reply:
Done